godtools/App/Services/Renderer/Views/MobileContent/Views/ContentText/MobileContentTextView.swift

```diff
@@ -86,14 +86,23 @@ class MobileContentTextView: MobileContentView {
     
     private func setupBinding(viewType: ViewType) {
         
+        let lineSpacing = CGFloat(2)
+        
         textLabel.backgroundColor = UIColor.clear
         textLabel.numberOfLines = 0
         textLabel.lineBreakMode = .byWordWrapping
         textLabel.font = viewModel.font
         textLabel.text = viewModel.text
         textLabel.textColor = viewModel.textColor
         textLabel.textAlignment = viewModel.textAlignment
-        textLabel.setLineSpacing(lineSpacing: 2)
+        textLabel.setLineSpacing(lineSpacing: lineSpacing)
+        
+        let lineHeight = viewModel.font.lineHeight
+        let minimumLines = viewModel.minimumLines
+        
+        let minimumHeight = minimumLines > 0 ? (lineHeight * minimumLines) + (lineSpacing * (minimumLines - 1)) : lineHeight
+        
+        textLabel.addHeightConstraint(constant: minimumHeight, relatedBy: NSLayoutConstraint.Relation.greaterThanOrEqual, priority: 1000)
```

Comment:
I can make minimumHeight optional.  As for lineSpacing, it looks like there's already a hardcoded value for it above, so I figured I would just apply that.  But I could remove it for now since it's possible that it is not consistent everywhere in the app.